build.gradle

```diff
@@ -30,12 +30,12 @@ subprojects {
             testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
             testCompile group: 'org.mockito', name: 'mockito-core', version: '2.10.0'
         }
-
-        compileJava {
-            sourceCompatibility = JavaVersion.VERSION_1_7
-            targetCompatibility = JavaVersion.VERSION_1_7
-            options.bootstrapClasspath = files("${System.env.JDK7_HOME}/jre/lib/rt.jar")
-        }
+//
+//        compileJava {
+//            sourceCompatibility = JavaVersion.VERSION_1_7
+//            targetCompatibility = JavaVersion.VERSION_1_7
+//            options.bootstrapClasspath = files("${System.env.JDK7_HOME}/jre/lib/rt.jar")
+//        }
```

Comment:
I don't have Java 1.7 installed.  We use Java 1.8 in our project.  Do we need to use Java 1.7 in order for this to work for us?